packages/client-web-kit-theme/assets/css/styles/realm.css

```
@@ -0,0 +1,186 @@
+    .a-realm-grid-item:focus-visible {
+        outline: none;
+        border-color: transparent;
+    }
+
+    .a-realm-grid-item:focus-visible::before {
+        opacity: 1;
+    }
+
+    .a-realm-grid-item-name {
+        font-weight: 600;
+        word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)